### ISSUE TITLE
"continuing this pattern" in output, instead of full data

### ISSUE THREAD

**Issue by bradmitchler** [run-llama/llama_parse#478]
I'm using Premium Mode to parse complex technical pdfs, with parsing instructions to break the table down by row with labels.  It does it properly for the first couple rows of each table, but then stops and just says "continuing this pattern..." instead of continuing and outputting the full table data.  

It actually has in the context a question from the LLM processing it "[Would you like me to continue with more detailed breakdowns of specific pressure points or flow rates for any particular model?]". I've tried editing the prompting it to always continue.

**Comment by BinaryBrain** [comment]
Hi @bradmitchler,
Can you please provide a JobID?

**Comment by bradmitchler** [comment]
Here's an example:
d4697f95-957d-415b-b192-5577fc286429

**Comment by BinaryBrain** [comment]
Looking at your prompt, I think you may have more consistent results by setting [is_formatting_instruction](https://docs.cloud.llamaindex.ai/llamaparse/features/parsing_options#is-formatting-instruction) to false, as you don't really want to change the format of the output but simply give some directions.

**Comment by bradmitchler** [comment]
I tried it - but then it doesn't break apart the data tables in to rows of text.  I'm specifically trying to modify the tables to display in rows of text, like the output of that job I sent.  It's doing it correctly, but not continuing for all cells / data points.  

If I turn off formatting instruction, it leaves the data tables fully in tact.  Thoughts?

**Comment by BinaryBrain** [comment]
Yeah, LlamaParse's purpose isn't really to reformat data. We have this feature because it's a nice hack for some use cases.
@hexapode I think this is a great use case for LlamaExtract
